hpicker.js

```diff
@@ -380,4 +381,4 @@ HorizontalPickerItem.defaultProps = itemDefaultProps;
 HorizontalPicker.propTypes = propTypes;
 HorizontalPicker.defaultProps = defaultProps;
 
-export default HorizontalPicker;
+export default HorizontalPicker;
```

Comment:
Looks good, but can you add a linefeed at the end of the file. Some linters complain about this.